idd/Energy+.idd.in

```diff
@@ -52999,9 +53001,13 @@ AirLoopHVAC:UnitarySystem,
        \type choice
        \key Load
        \key SetPoint
+       \key ASHRAE90VariableFan
```

Comment:
There are so many similar choices in the fan coil that ASHRAE90 is almost required to distinguish this from the other variable fan options.  But for the UnitarySystem, I like SingleZoneVAV.  The "Load" option works off the single control zone as well, correct?